Ontology/Capability/Sensor/PeopleCountSensor.json

```diff
@@ -11,7 +11,7 @@
         "en": "has value"
       },
       "name": "hasValue",
-      "schema": "integer",
+      "schema": "double",
```

Comment:
Why did we switch from integer to double for quantity of people?

Reply:
Good catch! That's from the underlying RealEstateCore OWL models. Apparently there are use cases where partial people counts are used, by some subsystems/sensors, and where such expressivity is needed. I think Per Karlberg could provide more detail if needed. I don't have the domain knowledge about such systems myself, but given that it is a more expressive schema, rather than a more constrained one, I figured it cannot hurt?

Reply:
Agreed. Can't hurt.